### PR TITLE
leaderboard page

### PR DESCRIPTION
**Pull request by abhikoreti**
Created table for leaderboard

### CODE REVIEW

File: adminpanel/urls.py [hunk 1]
```diff
@@ -15,8 +15,9 @@
 """
 from django.urls import path
 
-from adminpanel.views import index
+from adminpanel.views import index, home
```
Comment (pavankumarvm) [inline adminpanel/urls.py]: Replace index, home with *

File: adminpanel/urls.py [hunk 2]
```diff
@@ -15,8 +15,9 @@
...
 
 urlpatterns = [
   path('', index, name='index'),
+  path('home/',home,name='home'),
```
Comment (pavankumarvm) [inline adminpanel/urls.py]: Rename home with leaderboard

File: adminpanel/views.py [hunk 3]
```diff
@@ -2,4 +2,9 @@
 
 # Create your views here.
 def index(request):
-  return render(request, template_name='index.html')
\ No newline at end of file
+  return render(request, template_name='index.html')
+  
+def home(request):
```
Comment (pavankumarvm) [inline adminpanel/views.py]: Rename home with leaderboard

**Review by pavankumarvm — COMMENTED** [review]
Rename the home page as leaderboard page

File: adminpanel/views.py (hunk 3 above)
Comment (abhikoreti) [reply adminpanel/views.py]: Done in 77bd74d26ba5daf64e66c5f167aea9bd97787252

File: adminpanel/urls.py (hunk 2 above)
Comment (abhikoreti) [reply adminpanel/urls.py]: Done in 77bd74d26ba5daf64e66c5f167aea9bd97787252

File: adminpanel/urls.py (hunk 1 above)
Comment (abhikoreti) [reply adminpanel/urls.py]: Done in 77bd74d26ba5daf64e66c5f167aea9bd97787252

**Review by pavankumarvm — APPROVED** [review]
Done the required changes

**Pull request merged**